### PR TITLE
Additional fonts

### PR DESCRIPTION
**Pull request by rm-hull**
Fixes #226

### CODE REVIEW

File: luma/core/legacy/font.py [hunk 1]
```diff
@@ -22,6 +25,9 @@
 |                   | Spectrum.  Only contains characters 0x20-0x7E   |                                                                                                |
 |                   | inclusive; all others will appear as blanks.    |                                                                                                |
 +-------------------+-------------------------------------------------+------------------------------------------------------------------------------------------------+
+| ``SPECCY_FONT``   | Another font based on the character set from    | http://www.kreativekorp.com/software/fonts/index.shtml                                         |
```
Comment (thijstriemstra) [inline luma/core/legacy/font.py:28]: use https instead

File: luma/core/legacy/font.py [hunk 2]
```diff
@@ -8,6 +8,9 @@
 +-------------------+-------------------------------------------------+------------------------------------------------------------------------------------------------+
 | Font name         | Notes                                           | Source                                                                                         |
 +===================+=================================================+================================================================================================+
+| ``ATARI_FONT``    | Designed for a terminal emulator written for    | http://gnu.ethz.ch/linuks.mine.nu/atari/                                                       |
```
Comment (thijstriemstra) [inline luma/core/legacy/font.py:11]: this link doesn't resolve. But better to have something that nothing..?

File: luma/core/legacy/font.py [hunk 3]
```diff
@@ -1665,5 +1673,533 @@ def __getitem__(self, ascii_code):
     [0x00, 0x00, 0x00, 0x00],  # 0xFF
 ]
 
+#: Bit patterns for SPECCY_FONT
+#: (another font based on the character set from the ZX Spectrum).
+#: Source: http://www.kreativekorp.com/software/fonts/index.shtml,
```
Comment (thijstriemstra) [inline luma/core/legacy/font.py:1678]: use https instead

**Review by thijstriemstra — CHANGES_REQUESTED** [review]
See inline comments. I wouldn't use github username in pull request/commit message, as to not make an exception, or we'd have to do it for every contribution IMO.

File: luma/core/legacy/font.py (hunk 1 above)
Comment (rm-hull) [reply luma/core/legacy/font.py]: done

File: luma/core/legacy/font.py (hunk 2 above)
Comment (rm-hull) [reply luma/core/legacy/font.py:11]: yep, lets leave the link in just in case it ever came back

File: luma/core/legacy/font.py (hunk 3 above)
Comment (rm-hull) [reply luma/core/legacy/font.py]: done

**Review by thijstriemstra — DISMISSED** [review]

**Comment by thijstriemstra** [comment]
Ah, the link is broken, and detected by Sphinx: https://github.com/rm-hull/luma.core/runs/1657924751#step:8:192

I'll take a look.

**Review by thijstriemstra — CHANGES_REQUESTED** [review]
Fixed it but table is malformed now: https://github.com/rm-hull/luma.core/runs/1658072788#step:8:247

**Review by thijstriemstra — APPROVED** [review]

**Pull request merged**